packages/openhim-core-api/src/jembi-openhim-core-api.ts

```diff
@@ -0,0 +1,84 @@
+import axios from 'axios'
+
+/* ****************************************************************************************************************
+if running locally set REACT_APP_OPENHIM_API_BASE_URL environment variable to point to OpenHIM Core API base URL
+ **************************************************************************************************************** */
+
+if (!process.env.REACT_APP_OPENHIM_API_BASE_URL) {
+  throw new Error('REACT_APP_OPENHIM_API_BASE_URL is not set')
+}
+
+const API_URL =
+  process.env.REACT_APP_OPENHIM_API_BASE_URL || 'https://localhost:8080/'
```

Comment:
Shouldn't this be able to sort the check in L7? we already setting a default value if we can't get the environment variable. Otherwise, if we want to enforce the check the default URL will never be used.

Reply:
Yes, you are absolutely right, I fixed this issue yesterday locally, and I will push the fix with the next commit